pkg/project/admission/nodeenv/admission.go

```diff
@@ -26,8 +26,8 @@ type podNodeEnvironment struct {
 
 // Admit enforces that pod and its project node label selectors matches at least a node in the cluster.
 func (p *podNodeEnvironment) Admit(a admission.Attributes) (err error) {
-	// ignore anything except create or update of pods
```

Comment:
inline `*admission.Handler` and set to `Handler:    admission.NewHandler(admission.Create),`?